src/internal/api/server.go

```diff
@@ -0,0 +1,84 @@
+package api
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/darchlabs/nodes/src/internal/command"
+	"github.com/gofiber/fiber/v2"
+)
+
+type ServerConfig struct {
+	Port          string
+	CommandConfig *CommandConfig
+}
+
+type CommandConfig struct {
+	Chain            string
+	Runner           string
+	Host             string
+	DatabasePath     string
+	BootstrapNodeURL string
+}
+
+type Server struct {
+	server *fiber.App
+	cmd    *command.Command
+	port   string
+	chain  string
+}
+
+func NewServer(config *ServerConfig) *Server {
+	return &Server{
+		server: fiber.New(),
```

Comment:
The space format after keys and before values are strange here

Reply:
what do you mean with strange?